aten/src/ATen/core/TensorImpl.h

```diff
@@ -1486,17 +1471,29 @@ struct CAFFE2_API TensorImpl : public c10::intrusive_ptr_target {
 //    strong refcount           TODO: pack these into one word
 //    weak refcount
 //    storage pointer
-//    sizes vector (start)
-//    sizes vector (end)
-//    sizes vector (reserved)   TODO: get rid of me
-//    strides pointer
+//    sizes SmallVector (begin)
+//    sizes SmallVector (end)
+//    sizes SmallVector (reserved)
```

Comment:
Ugh, SmallVector has a reserved pointer? Totally useless for our use case XD

Reply:
Yes :/

Reply:
What's reserved mean here? I thought it was:

```
  void *BeginX, *EndX, *CapacityX;
```

Reply:
Oh as `std::vector::reserve`.